packages/core-datepicker/core-datepicker.js

```diff
@@ -1,6 +1,14 @@
 import { addStyle, closest, dispatchEvent, toggleAttribute, queryAll } from '../utils'
 import parse from '@nrk/simple-date-parse'
 
+const FILL = {
+  month: (self, value) => daysInMonth(self.parse(value)).filter(day => !self.disabled(day))[0] || value,
```

Comment:
I may be reading this wrong, but won't this always return the first available day in the month (except when every single day in the month is disabled)? If the current timestamp is 9 november and I switch to december using the month option I want 9 december to get selected, not 1 december.

Reply:
That might just be an expectation I have because that is the current behavior in most cases except for the current issue with disabled dates. Defaulting to the first available day in the month might be fine now that I've thought about it some more.

Reply:
```js
(self, value) =>
  !self.disabled(value)
    ? value
    : daysInMonth(self.parse(value)).filter(day => !self.disabled(day))[0] ||
      value
```

maybe keeping the current behaviour is best, except when the date is disabled?

Reply:
Yeah, only doing special logic when the date is disabled makes sense.